common/frame_session.go

```diff
@@ -737,7 +734,7 @@ func (fs *FrameSession) onPageNavigatedWithinDocument(event *cdppage.EventNaviga
 
 func (fs *FrameSession) onAttachedToTarget(event *target.EventAttachedToTarget) {
 	fs.logger.Debugf("NewFrameSession:onAttachedToTarget",
-		"sid:%v tid:%v esid:%v etid:%v ebctxid:%v type:%q",
+		"sid:%v tid:%v esid:%v etid:%v ebid:%v type:%q",
```

Comment:
Here too, `ebctxid` is more descriptive. It's an "event browser context ID", not an "event browser ID".